roles/dropwizard-api/README.md

```diff
@@ -9,6 +9,14 @@ The general process of deploying an API is
    /apis/config/API.yaml is present. Can either be installed
    ahead of time or copied along with the jar
 
+3. Write environment variables to /apis/env/API.env, in the form
+
+        PORT=8080
+        USER=admin
+        PASSWORD=!@#$^*"'
+
```

Comment:
How about if we mention that PATH is not allowed as a variable and will be ignored?